integration-tests/jcache/src/main/java/org/apache/camel/quarkus/component/jcache/it/JcacheResource.java

```diff
@@ -37,6 +37,9 @@ public class JcacheResource {
     @Inject
     CamelContext context;
 
+    @Inject
+    ProducerTemplate producerTemplate;
+
```

Comment:
Note that method like `loadComponentJcache` are generally removed as they don't provide that much value anymore in native mode.

Reply:
Done